### ISSUE TITLE
2019-10-11 check-in and to-do list

### ISSUE THREAD

**Issue by MelissaBradshaw** [MelissaBradshaw/ALLP#1]
Hello. I wanted to check in and let you all know what I've been doing over the Fall Break and set up some priorities for our next meeting on Tuesday.

@echristie111 and @cmccraw: 
1) I finished keywords for all but two of our sample letters. I still need to transcribe the letter to H.D. and the letter to Oppenheim that we'll be using. Once I type those up, I'll add the keywords in to our ALLP Sample Letter set spreadsheet. I created another spreadsheet for second string letters that we won't be using in this initial round. I know we will need a better organizing system for all of the letters, once we are past this sample stage, one that will be more permanent.

2) I started an ALLP Codebook in our Google Drive; as you start encoding the sample set, you can keep track of your decisions about what and how to tag there. Q: Would it be better to have the codebook live here in Git? I'm not sure that a codebook is that different from a Wiki. Let's talk about this on Tuesday, and let's go over a letter together and decide how to encode it.

3) Related to the codebook, I realize that I need to build a header sooner than later, and that it is a critical part of this early work. Let's check in about headers on Tuesday, as well. I'll have started to draft one, incorporating the participants and the funding we've had so far. It's okay for this to be minimal at this stage--the house slippers version of our meta-data--but I need to get something together.

**Zach**  Where are you? 

@xsun93: I look forward to seeing what you are thinking, in terms of design.

**Comment by cmccraw** [comment]
@MelissaBradshaw @echristie111 

I moved our "ALLP Encoding Sandbox" folder to the shared class folder— inside there is a folder for each sample letter, with the .doc of the transcription and .jpegs of the facsimile images from the archive. These are duplicates of the .doc transcription files from the "Sample Letters" folder, but I added "Sandbox" to the start of each filename to avoid any confusion!

Ericka and I are experimenting with ways of marking these up to get a sense of the scope of things to encode (alongside the spreadsheet, which is really the bulk of the encoding data!). We're meeting tomorrow to chat about anything else that might be within our encoding scope (materiality? etc.). The Sandbox is just a space for us to mark up and visualize the preexisting documents before starting to encode. 

https://github.com/cmccraw/LiliElbe_EngagedLearners ⬅️Here is the link to my Engaged Learners' class— much of this is customized to the workflow of the project (and is therefore quite elaborate!), but posting a link here in case it is helpful!

Can definitely factor in the TEI header sooner rather than later in our roadmap (forthcoming before next class!). & can talk about Codebook location on Tuesday!
